scrapers/PegasProductions.yml

```diff
@@ -0,0 +1,46 @@
+name: PegasProductions
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - pegasproductions.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+# To access other scenes go to Scene and select model which will show all their scenes
+      Title:
+        selector: //span[@itemprop="name"]
+      Date:
+        selector: //div[@id="date-duree"]/div[1]/p[1]
+        postProcess:
+          - parseDate: 02/01/2006
+      Performers:
+        Name: //div[@class="span10"]/p[1]/a
+      Tags:
+        Name:
+          selector: //div[@class="span10"]/p[2]
+          postProcess:
+            - replace:
+                - regex: "Tags:"
+                  with:
+          split: ","
```

Comment:
either use 
```yaml
          split: ", "
```
(note the extra space after the comma) or do a replace regex for `,\s` with `,` instead
The tags after the first get a leading space since `split` is the last action in the pp chain and the thus they wont be `cleaned` after the split.

Reply:
Should be all good to go now :-) , Thanks for the comments lets me learn all the time.